cmds/init/init.go

```diff
@@ -31,6 +33,95 @@ var (
 	debug   = func(string, ...interface{}) {}
 )
 
+// fillBuildBin tries to fill buildbin. It does it's level best but if it can not do it,
+// c'est la vie.
+func fillBuildBin() {
+	var (
+		// mark that we visited a directory, remember the name
+		// of the go file in that directory. If the name is "",
+		// we won't bother to test it.
+		visited = map[string]string{
+			"/src/github.com/u-root/u-root/cmds/installcommand": "",
+			"/src/github.com/u-root/u-root/cmds/init":           "",
+			"/src/github.com/u-root/u-root/cmds/rush":           "",
+		}
+		// populate is the list of things we must populate.
+		// If you know something you just gotta have, then put it here.
+		populate = []string{"rush"}
+	)
+
+	// populate buildbin
+	f, err := find.New(func(f *find.Finder) error {
+		f.Pattern = "*.go"
+		f.Root = "/src"
+		return nil
+	})
+	if err != nil {
+		log.Printf("Can't build a find, sorry, err %v", err)
+		return
+	}
+	go f.Find()
+
+	for goFile := range f.Names {
+		if goFile.Err != nil {
+			log.Printf("%s: got %v, want nil", goFile.Name, goFile.Err)
+			continue
+		}
+		dir := filepath.Dir(goFile.Name)
+		if _, ok := visited[dir]; ok {
+			continue
+		}
+		visited[dir] = goFile.Name
+	}
+	pm, err := regexp.Compile("\npackage  *main")
```

Comment:
Won't the leading `\n` skip commands files that start with `package main` on line 1?

Reply:
yeah, this is stupid. This approach is not workable.